src/Entities/Conditions/AllOfCondition.php

```diff
@@ -25,7 +25,7 @@ public function check(TargetContract $target, StateContract $state): bool
             if ($conditionResult !== $this->expectedResult()) {
                 return false;
             }
-            $actions = \array_merge($actions, $condition->getActions());
+            $actions = \array_merge($actions, \collect($condition->getActions())->toArray());
```

Comment:
can you just cast it to array?

Reply:
cast is not working correctly:
![image](https://user-images.githubusercontent.com/5069097/54205647-cd0fa700-44e7-11e9-9f33-49f1a9a24d9b.png)